packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {
+  css,
+  autobind,
+  BaseComponent
+} from '../../Utilities';
+import { IButtonProps } from '../../Button';
+import { IOverflowSetProps } from './OverflowSet.Props';
+import { FocusZone, FocusZoneDirection } from '../../FocusZone';
+
+import * as stylesImport from './OverflowSet.scss';
+const styles: any = stylesImport;
+
+export class OverflowSet extends BaseComponent<IOverflowSetProps, null> {
+
+  public render() {
+    let {
+      items,
+      overflowItems,
+      onRenderOverflowButton
+    } = this.props;
+
+    const overflowButtonProps: IButtonProps = {
+      menuProps: { items: overflowItems }
+    };
+
+    return (
+      <FocusZone className={ css('ms-OverflowSet', styles.root) } direction={ FocusZoneDirection.horizontal } role='menubar' >
+        { items && this._onRenderItems(items) }
+        { overflowItems.length && onRenderOverflowButton(overflowButtonProps) }
+      </FocusZone>
+    );
+  }
+
+  @autobind
+  private _onRenderItems(items: any[]): JSX.Element[] {
+    return items.map((item, i) => {
+      let key = item.key ? item.key : i;
```

Comment:
The props definition should say that items can take a key. It might even be worth explicitly stating this in the type definition instead of just putting any (though not sure how to best do that, maybe an intersection type?